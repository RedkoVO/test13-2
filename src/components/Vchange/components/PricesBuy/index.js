import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import styles from './styles'

const PricesBuy = ({ classes, pricesBuyData }) => (
  <div className={classes.root}>
    <div className={classes.title}>
      <span>Buy</span> BTC: Prices
    </div>

    <table className={classes.wrTable} cellSpacing="0">
      <tbody>
        <tr>
          <th>Price (USDT)</th>
          <th>Ammount (BTC)</th>
          <th>Total (USDT)</th>
        </tr>
        {pricesBuyData.map(item => (
          <tr className={classes.wrItem} key={item.id}>
            <td>{item.price}</td>
            <td>{item.ammount}</td>
            <td>{item.total}</td>
          </tr>
        ))}
      </tbody>
    </table>
  </div>
)

PricesBuy.propTypes = {
  pricesBuyData: PropTypes.array
}

export default withStyles(styles)(PricesBuy)
