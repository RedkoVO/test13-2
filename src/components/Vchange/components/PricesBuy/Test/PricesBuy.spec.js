import * as React from 'react'
import renderer from 'react-test-renderer'

import PricesBuy from '../'

const pricesBuyData = [
  { id: 1, price: '3752.22', ammount: '1.0000', total: '3752.2200000' },
  { id: 2, price: '3752.22', ammount: '1.0000', total: '3752.2200000' },
  { id: 3, price: '3752.22', ammount: '1.0000', total: '3752.2200000' },
  { id: 4, price: '3752.22', ammount: '1.0000', total: '3752.2200000' }
]

it('<PricesBuy /> renders correctly', () => {
  const tree = renderer
    .create(<PricesBuy pricesBuyData={pricesBuyData} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
