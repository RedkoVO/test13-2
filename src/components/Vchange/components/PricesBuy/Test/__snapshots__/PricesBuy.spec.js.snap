// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PricesBuy /> renders correctly 1`] = `
<div
  className="PricesBuy-root-1"
>
  <div
    className="PricesBuy-title-2"
  >
    <span>
      Buy
    </span>
     BTC: Prices
  </div>
  <table
    cellSpacing="0"
    className="PricesBuy-wrTable-3"
  >
    <tbody>
      <tr>
        <th>
          Price (USDT)
        </th>
        <th>
          Ammount (BTC)
        </th>
        <th>
          Total (USDT)
        </th>
      </tr>
      <tr
        className="PricesBuy-wrItem-4"
      >
        <td>
          3752.22
        </td>
        <td>
          1.0000
        </td>
        <td>
          3752.2200000
        </td>
      </tr>
      <tr
        className="PricesBuy-wrItem-4"
      >
        <td>
          3752.22
        </td>
        <td>
          1.0000
        </td>
        <td>
          3752.2200000
        </td>
      </tr>
      <tr
        className="PricesBuy-wrItem-4"
      >
        <td>
          3752.22
        </td>
        <td>
          1.0000
        </td>
        <td>
          3752.2200000
        </td>
      </tr>
      <tr
        className="PricesBuy-wrItem-4"
      >
        <td>
          3752.22
        </td>
        <td>
          1.0000
        </td>
        <td>
          3752.2200000
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
